### ISSUE TITLE
Adding an emoji to a field layout Tip element causes broken JSON in `craft_fieldlayouttabs`

### ISSUE THREAD

**Issue by mmikkel** [craftcms/cms#7677]
### Description

Adding an emoji to a field layout Tip element and saving the field layout causes broken JSON in the `elements` column in the relevant `craft_fieldlayouttabs` table. Specifically, it appears that the JSON string is simply cut off, from where the emoji would be. See example broken JSON [here](https://pastebin.com/bUY7yd25).

Since the JSON is now invalid, attempting to render the relevant edit page for the field layout triggers an `InvalidArgumentException` (example stack trace for that error is [here](https://pastebin.com/5SX9sUj9)).

### Steps to reproduce

1. Add a Tip element to a field layout, and add an emoji like 😉 to it
2. Save the field layout. It should save without any errors or exceptions
3. Navigate to the edit page using that layout. Get an `InvalidArgumentException`
4. Confirm that the JSON in the `craft_fieldlayouttabs` table is now truncated and invalid

### Additional info

Only tested locally on macOS 10.14.6 using Valet.

- Craft version: 3.6.10
- PHP version: 7.4.13
- Database driver & version: 5.7.32
- Plugins & versions:

* Child Me! 1.2.0
* CP Clear Cache 1.1.0
* CP Field Inspect 1.2.4
* Image Resizer 2.1.0
* Imager X v3.3.0
* Redactor 2.8.5
* SEOMate 1.1.9

**Comment by brandonkelly** [comment]
I’m not able to reproduce this. I get the following SQL error when attempting to save the field layout:

> General error: 1366 Incorrect string value: '\xF0\x9F\x98\x89",...' for column 'elements' at row 1

…which is expected when saving a 4+ byte character to a MySQL table that isn’t using the utif8mb4 charset.

You’re not seeing a similar error though?

**Comment by mmikkel** [comment]
No, there's no error on my end when saving the field layout. Just tested it again with a fresh Craft install.

**Comment by brandonkelly** [comment]
What charset is your `fieldlayouttabs` table set to?

**Comment by mmikkel** [comment]
`utf8`
